go.mod

```diff
@@ -54,6 +55,7 @@ require (
 	github.com/thoas/go-funk v0.9.1
 	github.com/tidwall/gjson v1.10.2
 	github.com/tidwall/sjson v1.0.4
+	github.com/valyala/fastjson v1.6.3
```

Comment:
After we reach to a conclusion regarding which approach we are going to follow and before merging this, we shall cleanup the dependencies here.
If we need to keep the benchmarks for posterity, we shall do so in a separate module, either here or in [rudder-tests](https://github.com/rudderlabs/rudder-tests)

Reply:
Yes I totally agree. Before merging I'll remove everything. I was thinking maybe a link to this PR for posterity in the `getUsersPayload` would be enough.